src/builder/index.js

```diff
@@ -14,18 +13,42 @@ export default function(options) {
     config,
   } = options
 
+  // Prepare staticAssets path and route
+  let { staticAssets } = options
+  if (staticAssets !== undefined) {
+    if (typeof staticAssets === "string") {
+      staticAssets = {
+        path: path.join(config.cwd, config.source, staticAssets),
+        route: staticAssets,
+      }
+    } else if (
```

Comment:
@MoOx how to fix this lint rule ?

line 24:12 `Closing curly brace appears on the same line as the subsequent block`